src/components/customerCases/customerCase/contactInformation/ContactInformation.tsx

```diff
@@ -54,21 +54,30 @@ export default async function ContactInformation({
   );
   const t = await getTranslations("contact_information");
   return (
-    <div className={styles.wrapper}>
-      <div className={styles.content}>
-        <div className={styles.titleSection}>
-          <Text type={"bodyXl"}>{t("help")}</Text>
-          <Text type="bodyBig">{t("contact_sale")} </Text>
+    <>
+      <section className={styles.contactBox}>
+        <div
+          className={`${styles.contactBox__inner} ${styles["contactBox__inner--light"]}`}
+        >
+          <div className={styles.textContent}>
+            <Text type="h3" as="h2">
+              {t("help")}
+            </Text>
+            <Text type="bodyBig">{t("contact_sale")} </Text>
+          </div>
+          <div className={styles.contactSelectorWrapper}>
+            <Suspense fallback={<EmployeeCardSkeleton background={"light"} />}>
+              <ContactSelector
+                language={language}
+                locations={locationsWithContact}
+                contactByLocation={contactByLocation}
```

Comment:
This shouldn't be awaited, but left a promise to utilise streaming and lazy loading of content (quicker load time)

Reply:
What do you mean? Should I not use Suspense?